ConanToolWindowControl.xaml.cs

```diff
@@ -434,7 +434,12 @@ private void Configuration_Click(object sender, RoutedEventArgs e)
         private void ShowPackages_Click(object sender, RoutedEventArgs e)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
+            Project startupProject = ProjectConfigurationManager.GetStartupProject(_dte);
+            string projectFilePath = startupProject.FullName;
+            string projectName = startupProject.Name;
+            string projectDirectory = Path.GetDirectoryName(projectFilePath);
 
+            MessageBox.Show(string.Join("\n", GetConandataRequirements(projectDirectory)), $"Installed packages for '{projectName}' - Conan C/C++ Package Manager");

```

Comment:
Just so we don't forget, it would be nice to do a GetConandataRequirements(project)